client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -145,7 +145,7 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 					return reduxDispatch(
 						recordTracksEvent( 'calypso_checkout_composite_stored_card_error', {
 							error_type: action.payload.type,
-							error_message: String( action.payload.message ),
```

Comment:
Nice. I actually think this was a bug with the previous implementation.

Reply:
Yes, it's possible that this particular error has never come up.